File: Programing_Basics_with_JS/firstStepInCodingMoreExercises/08.CircleAreaAndPerimeter.js
function circleAreaAndPerimeter(input) {
    let radius = Number(input[0]);

    let circleArea = radius * radius * Math.PI;
    let perimeter = radius * 2 * Math.PI;

    console.log(circleArea.toFixed(2));
    console.log(perimeter.toFixed(2));
}

circleAreaAndPerimeter(["3"])
circleAreaAndPerimeter(["4.5"])

File: Programming_Basics_with_Python/first_step_in_coding_more_exercise/08._circle_area_and_perimeter.py
from math import pi


def circle_area(r):
    return f'{pi * r**2:.2f}'


def circle_perimeter(r):
    return f'{2 * (pi * r):.2f}'


r = float(input())

print(circle_area(r))
print(circle_perimeter(r))
